prow/github/fakegithub/fakegithub.go

```diff
@@ -358,3 +362,10 @@ func (f *FakeClient) ListMilestones(org, repo string) ([]github.Milestone, error
 	}
 	return milestones, nil
 }
+
+// ListCommits lists milestones.
+func (f *FakeClient) ListCommits(org, repo string, prNumber int) ([]github.RepositoryCommit, error) {
+	// TODO: should we make a copy of the slice before returning it?
```

Comment:
not sure if needed to copy.
leaning towards "no" AKA make the callers responsible.